### PR TITLE
fix: Added support for odata bindings in create and update

### PR DESCRIPTION
**Pull request by magesoe**

### CODE REVIEW

**Comment by thygesteffensen** [comment]
It needs a version of [XrmMockup](https://github.com/delegateas/XrmMockup) that does not exist yet, follow https://github.com/delegateas/XrmMockup/pull/222.

**Comment by magesoe** [comment]
New XrmMockup version has been pushed

File: PAMU_CDS/PamuCdsDependencyExtension.cs [hunk 1]
```diff
@@ -31,9 +31,6 @@ public static void AddPamuCds(this IServiceCollection services)
 
             services.AddFlowActionByApiIdAndOperationsName<DisAndAssociateEntitiesAction>(apiId,
                 DisAndAssociateEntitiesAction.OperationId);
-
-            services.AddFlowActionByApiIdAndOperationsName<ScopeActionExecutor>(apiId,
-                new[] {"ExecuteChangeset"});
         }
     }
 }
```
Comment (thygesteffensen) [inline PAMU_CDS/PamuCdsDependencyExtension.cs:36]:
```suggestion
}

```

File: Test/PureCdSceFullFlowTest.cs [hunk 2]
```diff
@@ -20,5 +21,22 @@ public void TestCreateContact()
...
+            {
+                ColumnSet = new ColumnSet(true)
+            }).Entities.FirstOrDefault();
+
+            Assert.IsNotNull(retrievedContact);
+            Assert.AreEqual(account.Id, retrievedContact.GetAttributeValue<EntityReference>("parentcustomerid")?.Id);
+
+        }
     }
 }
```
Comment (thygesteffensen) [inline Test/PureCdSceFullFlowTest.cs:42]:
```suggestion
}

```

File: Test/TestFlows/LookupFlow.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            },
+            "authentication": "@parameters('$authentication')"
+          }
+        }
+      }
+    },
+    "templateName": ""
+  },
+  "schemaVersion": "1.0.0.0"
+}
```
Comment (thygesteffensen) [inline Test/TestFlows/LookupFlow.json:74]:
```suggestion
}

```

**Review by thygesteffensen — APPROVED** [review]
💨 🚀

**Pull request merged**